src/databricks/labs/remorph/config.py

```diff
@@ -104,6 +104,13 @@ class ReconcileTablesConfig:
     tables_list: list[str]  # [*, table1, table2]
 
 
+@dataclass
+class ReconcileMetricsConfig:
```

Comment:
Can we rename the dataclass as Recondbconfig or Reconmetadataconfig

Reply:
`ReconcileMetadataConfig`